README.md

```diff
@@ -43,6 +43,8 @@ Here's a summary of the available configuration options:
 | relay.environmentVariables                    | array   | `[]`                                                         | Defines container environment variables to configure the Relay Proxy instance (via container spec) |
 | relay.secrets                                 | array   | `[]`                                                         | Defines container environment variables or volumes populated from k8s secrets                      |
 | relay.volume                                  | object  | `{}`                                                         | Enables offline mode or references an existing config file from a defined volume                   |
+| relay.livenessProbe                           | object  | `{httpGet: { port: "api",  path: "/status" }}`               | Defines the liveness probe for the relay container                                                 |
```

Comment:
Do we need documentation that the scheme can be changed, or this is standard for httpGet already so developers would already know what to do?

Reply:
httpGet is a defined liveness configuration in k8s. So I think it will be well-known.